examples/blender/blender.py

```diff
@@ -52,18 +56,25 @@ async def worker(ctx: WorkContext, tasks):
         max_workers=3,
         budget=10.0,
         timeout=init_overhead + timedelta(minutes=len(frames) * 2),
-        subnet_tag="testnet",
+        subnet_tag=subnet_tag,
     ) as engine:
 
         async for progress in engine.map(worker, [Task(data=frame) for frame in frames]):
             print("progress=", progress)
 
+def build_parser():
+    parser = argparse.ArgumentParser(description='Render blender scene')
+    parser.add_argument('--subnet-tag', default='testnet')
+    parser.add_argument('--debug', dest='log_level', action='store_const', const=logging.DEBUG, default=logging.INFO)
+    return parser
 
 if __name__ == "__main__":
+    parser = build_parser()
```

Comment:
Please compare with:
https://github.com/golemfactory/yapapi/pull/35/files
Both branches are similar.

Reply:
Thanks for pointing me to #35 . I prefer generating parser than returning parsed args, because parser could be extended. It's especially important when this builder is shared between examples.